substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/BundleSupport.java

```diff
@@ -849,7 +995,7 @@ private void write() {
             boolean imageBuilt = !nativeImage.isDryRun();
             properties.put(PROPERTY_KEY_IMAGE_BUILT, String.valueOf(imageBuilt));
             if (imageBuilt) {
-                properties.put(PROPERTY_KEY_BUILT_WITH_CONTAINER, String.valueOf(false));
+                properties.put(PROPERTY_KEY_BUILT_WITH_CONTAINER, String.valueOf(useContainer));
```

Comment:
Currently this is a write-only property. But it should not be. If a bundle was created with `,container` we have to make a rebuild of that bundle also happen in a container without requiring the user to specify the `,container` extra arguments explicitly. I.e. the `,container` is sticky for bundle builds (Once built inside container, always build inside container).

Reply:
Makes sense, but should there then also be a way to force the bundle to not use a container?

Reply:
Not really. If the the bundle has PROPERTY_KEY_BUILT_WITH_CONTAINER set to true and platform we are trying to rebuild on has container support then we should rebuild inside the container.

But we should print a warning if a bundle with PROPERTY_KEY_BUILT_WITH_CONTAINER set gets rebuilt on a platform where there is no container support (i.e. Windows or Linux with neither podman or docker)

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/18bfbe320ba58354454f78db8739015b19c853ff